style/pileup.css

```diff
@@ -24,7 +60,7 @@
   text-anchor: middle;
 }
 .pair.loose text {
-  font-size: 28;
+  font-size: 24;
```

Comment:
Usually that would be true, but in this case I'm styling an SVG element. SVG has its own notion of coordinates, which are unit-free. This corresponds to 24px.